releases.md

```diff
@@ -41,6 +41,13 @@ Further documentation available:
 
 ## Release
 
+### v0.45
+
+- **Latest Release**: [v0.45.0][v0-45-0] (2023-02-17) ([docs][v0-45-0-docs], [examples][v0-45-0-examples])
+- **Initial Release**: [v0.45.0][v0-45-0] (2023-02-17)
+- **End of Life**: 2023-06-17
```

Comment:
we're only supporting new releases other than LTS for 1 month now

Reply:
Fixed!